specification/mediaservices/resource-manager/Microsoft.Media/preview/2018-03-30-preview/StreamingPoliciesAndStreamingLocators.json

```diff
@@ -0,0 +1,1248 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "streamingLocatorName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Streaming Locator name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Delete a Streaming Locator": {
+            "$ref": "examples/streaming-locators-delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/streamingLocators/{streamingLocatorName}/listContentKeys": {
+      "post": {
+        "summary": "List Content Keys",
+        "description": "List Content Keys used by this Streaming Locator",
+        "operationId": "StreamingLocators_ListContentKeys",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ListContentKeysResponse"
+            }
+          },
+          "404": {
+            "description": "NotFound"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "streamingLocatorName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Streaming Locator name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "List Content Keys": {
+            "$ref": "examples/streaming-locators-list-content-keys.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/streamingLocators/{streamingLocatorName}/listPaths": {
+      "post": {
+        "summary": "List Paths",
+        "description": "List Paths supported by this Streaming Locator",
+        "operationId": "StreamingLocators_ListPaths",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ListPathsResponse"
+            }
+          },
+          "404": {
+            "description": "NotFound"
+          },
```

Comment:
Unsuccessful status codes should not be included

Reply:
In this case, I agree we shouldn't have 404 here and I removed it.

Reply:
Fixed